lib/library.rb

```diff
@@ -0,0 +1,69 @@
+require 'yaml'
+require 'date'
+require 'pry'
+require './lib/person.rb'
+
+class Library
+
+attr_accessor :items
+
+def initialize
+  @items = YAML.load_file('./lib/data.yml')
+end
+
+def check_out(title, person)
+ case
+ when
+   if_not_available?(title, person) then 'Book is not available'
+ else
+   perform_checkout(title, person)
+ end
+end
```

Comment:
See if you can't refactor this not to use a `case` statement. Case statements are switches for a variables. So if you have a variable `animal` and you are setting it somewhere and you want to know the `weight`, your `case` statement would look like:
```
case animal
when 'moose'
  weight = 7000
when 'dog'
  weight = 40
end
```
Feel me? So here it's not appropriate to use a `case` statement because we're not utilizing its structure. A couple of `if` statements probably makes more sense. Make sure you test your refactor.